README.rst

```diff
@@ -5,7 +5,10 @@ VAGRANT BOXES
 REQUIREMENTS
 ============
 
-* Hashicorp packer (will be installed automatically).
```

Comment:
Travis will automatically install packer. But the normal user still has to install packer right?

Reply:
Still under developing ;-)